.github/actions/deploy-klt-on-cluster/action.yml

```diff
@@ -56,21 +51,15 @@ runs:
         echo "Importing image: $image"
         kind load image-archive $image/$image -n ${{ inputs.cluster-name }}
       done
-  
-  - name: Install cert-manager
-    shell: bash
-    run: |
-      kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${{ inputs.cert-manager-version }}/cert-manager.yaml
-      kubectl rollout status deployment cert-manager-cainjector -n cert-manager -w
-      kubectl rollout status deployment cert-manager-webhook -n cert-manager -w
-      kubectl rollout status deployment cert-manager -n cert-manager -w
     
   - name: Install lifecycle-toolkit
     shell: bash
     run: |
       sed -i 's/imagePullPolicy: Always/imagePullPolicy: Never/g' ~/download/artifacts/keptn-lifecycle-operator-manifest-test/release.yaml
       sed -i 's/ghcr.keptn.sh\/keptn\/functions-runtime:.*/localhost:5000\/keptn\/functions-runtime:${{ inputs.functions_runtime_tag }}/g' ~/download/artifacts/keptn-lifecycle-operator-manifest-test/release.yaml
+      kubectl apply -f ~/download/artifacts/keptn-cert-manager-manifest-test
```

Comment:
don't we need some time before deploying KLT to get the cert manager up and running?

Reply:
no clue, seems to work fine as is...